modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/notifications/NotificationsInfo.js

```diff
@@ -387,21 +389,21 @@ const NotificationsInfo = ({
 				if (
 					previousItem.data.notifications.recipients[
 						notificationIndex
-					]
+					]?.[0]
```

Comment:
@rebsilva This `?.` was not in your previous review. Had to add it to fix a bug found by 3 Poshi tests:

https://testray.liferay.com/home/-/testray/case_results/2565672159
https://testray.liferay.com/home/-/testray/case_results/2565672105
https://testray.liferay.com/home/-/testray/case_results/2565672192

The code was basically trying to access the position [0] in the array before it actually existed, which crashed the application.

Reply:
okay! i will test it again just to make sure everything is working, thanks!